src/components/Messages/CreateMessageModal/MessagesModalList.tsx

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { List } from "@mui/material";
+import { useAppSelector } from "../../../state/hooks";
+import MessagesModalListItem from "./MessagesModalListItem";
+
+export type OtherUser = {
+  otherUserId: number;
+  displayName: string;
+  username: string;
+};
+
+type MessagesListProps = {
+  onClose: () => void;
+};
+
+const MessagesList = ({ onClose }: MessagesListProps) => {
+  const user = useAppSelector((state) => state.user);
+  const [conversationList, setConversationList] = useState<OtherUser[]>([]);
+
+  useEffect(() => {
+    const fetchConversationList = async () => {
+      const result = await axios.get(
+        "http://localhost:3001/api/messages/getModalConversations",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setConversationList(result.data as OtherUser[]);
+    };
+    fetchConversationList();
+  }, [user]);
+
+  return (
+    <List component="div">
+      {conversationList.map((o, index) => (
+        <MessagesModalListItem otherUser={o} key={index} onClose={onClose} />
```

Comment:
Do you think we can use the other user's id as the key? You can only have one conversation with each unique user, so I think it makes sense

Reply:
yep this makes sense